test/parallel/test-cluster-bind-privileged-port.js

```diff
@@ -21,6 +21,18 @@
 
 'use strict';
 const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const net = require('net');
+const { execSync } = require('child_process');
+
+if (common.isLinux) {
+  const sysctlOutput = execSync('sysctl net.ipv4.ip_unprivileged_port_start').toString();
+  const unprivilegedPortStart = parseInt(sysctlOutput.split(' ')[1], 10);
```

Comment:
Still using the wrong index. :-)

`parseInt('=')` returns `NaN` and that's why `unprivilegedPortStart <= 42` is always false now.

Reply:
Fixed hopefully :)